packages/sit-onyx/src/styles/grid.scss

```diff
@@ -126,4 +189,75 @@ $gridVariants: (
       }
     }
   }
+
+  // Sidebar
+
+  .onyx-sidebar {
+    container-type: inline-size;
+
+    .onyx-grid {
+      display: grid;
+      grid-template-columns: repeat(var(--onyx-sidebar-grid-columns), minmax(0, 1fr));
+      gap: var(--onyx-sidebar-grid-gutter);
+    }
+    .onyx-grid-container {
+      // Note: The --onyx-grid-margin is implemented as padding here so that is also included in e.g. custom background colors
+      // but the term "margin" comes from UX/Figma so we decided to keep naming it margin to be aligned
+      padding: var(--onyx-sidebar-grid-margin);
+      max-width: var(--onyx-sidebar-grid-max-width, 100%);
+      margin-inline: var(--onyx-sidebar-grid-margin-inline);
+      box-sizing: border-box; // max-width should include the padding
+      transition: max-width 1s;
+    }
+    /**
+* Grid element class for defining how many columns the element spans
+*/
+
+    @for $i from 1 through 12 {
+      .onyx-grid-span-#{$i} {
+        grid-column-end: span min($i, var(--onyx-sidebar-grid-columns));
+      }
+    }
+    /**
+* Grid element class for spanning the full row width.
+*/
+
+    .onyx-grid-span-full {
+      grid-column: 1 / span var(--onyx-sidebar-grid-columns);
+    }
+  }
```

Comment:
We shouldn't need to override/copy+paste all of this. In the generated properties below, you can just use the same variable names as for the regular grid so they will be overriden:

e.g. `onyx-grid-columns` instead of `onyx-sidebar-grid-columns`

Reply:
So imo the only variables we need to set/override here are:
```css
--onyx-grid-columns
--onyx-grid-gutter
--onyx-grid-margin
```